src/bundles/AiDrawer/AiDrawer.tsx

```diff
@@ -173,16 +171,11 @@ type AiDrawerProps = {
    */
   fetchOpts?: AiChatProps["requestOpts"]["fetchOpts"]
 
-  /**
-   * Pass to target a specific drawer instance where multiple are on the page.
-   */
-  /** @deprecated The AiDrawerManager now handles multiple AiDrawer instance removing the need to target */
-  target?: string
-
-  payload?: AiDrawerInitMessage["payload"]
+  settings?: AiDrawerSettings
```

Comment:
I changed `payload` to `settings`.

"payload" makes sense for the component that handles `window.postMessage` events (the AiDrawerManager) but IMO doesn't really make sense for the drawer itself, which doesn't care about the events.